ipam/ipv6Ipam.go

```diff
@@ -211,7 +212,9 @@ func (source *ipv6IpamSource) refresh() error {
 		return err
 	}
 
+	source.isLoaded = true
```

Comment:
this is subject to a race, are the side effects to be concerned about if this fn executes more than once?

Reply:
this function is getting called within mutex lock.  is it still a concern?